### PR TITLE
fix: poweralert ups divisor

### LINKED ISSUE #4391: TrippLite SNMPWEBCARD values off by 10x
DO NOT DELETE THIS INFORMATION.

> Please read this information carefully.

- [x] Is your install up to date? [Updating your install](http://docs.librenms.org/General/Updating/)
      Please do not submit an issue if your install is not up to date within the last 24 hours or on a stable monthly release.
- [ ] Please include all of the information between the `====================================` section of `./validate.php`.
- [ ] If you would like us to add a new device then please provide the information asked for [here](http://docs.librenms.org/Support/FAQ/#faq20)
- [ ] Please provide as much detail as possible.

SNMPWEBCARD with version 12.06.0068.11852.11851 reports incorrect values for battery voltage, output current, input frequency, output frequency and bypass frequency. Values are 10x what they should be. 

here is the snmpbulkwalk output
http://pastebin.com/JMAzn8J8

### PR DESCRIPTION
**Pull request by crcro**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

fix #4391

### CODE REVIEW

**Comment by laf** [comment]
I've updated this PR to centralise the code a bit more.

@librenms/reviewers can someone else now check this pls.

File: includes/discovery/sensors/pre-cache/poweralert.inc.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ *
+ * Copyright (c) 2016 Neil Lathwood <neil@lathwood.co.uk>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+if ($device['os'] == 'junos') {
```
Comment (crcro) [inline includes/discovery/sensors/pre-cache/poweralert.inc.php]: maybe a type as os should be poweralert?

Comment (laf) [inline includes/discovery/sensors/pre-cache/poweralert.inc.php]: Well spotted - updated.

File: includes/discovery/sensors/voltages/rfc1628.inc.php [hunk 2]
```diff
@@ -16,9 +16,15 @@
             $volt_id          = $split_oid[(count($split_oid) - 1)];
             $volt_oid         = "1.3.6.1.2.1.33.1.2.5.$volt_id";
             $divisor          = 10;
-            if ($device['os'] == 'poweralert' || $device['os'] == 'poweralert') {
+
+            if ($device['os'] == 'netmanplus') {
```
Comment (crcro) [inline includes/discovery/sensors/voltages/rfc1628.inc.php]: shouldn't this be using newly generate function? just to standardize things?

Comment (laf) [inline includes/discovery/sensors/voltages/rfc1628.inc.php]: It's your PR :)

**Pull request closed without merging**